android/flavors/gms-enabled/app/organicmaps/location/GoogleFusedLocationProvider.java

```diff
@@ -59,6 +62,7 @@ public void onLocationAvailability(@NonNull LocationAvailability availability)
     super(listener);
     mFusedLocationClient = LocationServices.getFusedLocationProviderClient(context);
     mSettingsClient = LocationServices.getSettingsClient(context);
+    mContext = context;
```

Comment:
Can context change at some point later, so this stored one won't work properly?

Reply:
It is the same context which is in LocationHelper (MwmApplication).